docs/root/intro/version_history.rst

```diff
@@ -32,6 +32,7 @@ Version history
 * grpc-json: added support for :ref:`ignoring unknown query parameters<envoy_api_field_config.filter.http.transcoder.v2.GrpcJsonTranscoder.ignore_unknown_query_parameters>`.
 * grpc-json: added support for :ref:`the grpc-status-details-bin header<envoy_api_field_config.filter.http.transcoder.v2.GrpcJsonTranscoder.convert_grpc_status>`.
 * header to metadata: added :ref:`PROTOBUF_VALUE <envoy_api_enum_value_config.filter.http.header_to_metadata.v2.Config.ValueType.PROTOBUF_VALUE>` and :ref:`ValueEncode <envoy_api_enum_config.filter.http.header_to_metadata.v2.Config.ValueEncode>` to support protobuf Value and Base64 encoding.
+* http: added a default 1 hour idle timeout to upstream and downstream connections. HTTP connections with no stream and no activity will be closed after 1 hour unless the default is overridden.
```

Comment:
sorry, forgot this before but can you ref link to the relevant idle timeout and/or protocol fields in the HCM/cluster so people know how to set this?